### ISSUE TITLE
ZWave Can't Update

### ISSUE THREAD

**Issue by guarddog13** [tteck/Proxmox#603]
**Describe the issue**
Tried to update Zwave JS as HA is giving a repair for an update. The update script fails and Zwave JS has to be rebooted to get it back.  

**Bug reports will be closed after 12 hours of NO activity.**

**Comment by tteck** [comment]
Need a little more info.

**Comment by guarddog13** [comment]
![Screenshot (9)](https://user-images.githubusercontent.com/31947215/194186315-7fa2c5cc-0222-4288-8cb2-ec4d60afa28a.png)

![Screenshot (10)](https://user-images.githubusercontent.com/31947215/194186417-bd4c4105-18a7-41e2-b4c1-e36aada479b2.png)

![Screenshot (11)](https://user-images.githubusercontent.com/31947215/194186435-f640f6d8-2449-4259-9265-e561e358d02a.png)

**Comment by guarddog13** [comment]
![Screenshot (12)](https://user-images.githubusercontent.com/31947215/194186608-3dfb693c-b097-4738-9143-619f18414b73.png)

**Comment by tteck** [comment]
working on it

**Comment by tteck** [comment]
remove the files that was downloaded
then run the update script again

**Comment by guarddog13** [comment]
> remove the files that was downloaded

I don't' follow...what files?

**Comment by tteck** [comment]
hold on, they changed a bunch of stuff.

**Comment by guarddog13** [comment]
Yeah I can't even load a new instance or get to my UI now.

**Comment by tteck** [comment]
Man, what a cluster. They renamed everything again to make way for what looks like a Windows version.
Anyway, I got it all worked out. 
In your LXC console, run `cd /opt/zwave-js-ui` then `ls` and remove **any** `.zip` file 
(`rm zwave-js-ui-v8.1.0-linux.zip zwave-js-ui-v8.1.0-win.zip`) 
Then run the update script

**Comment by guarddog13** [comment]
![Screenshot (13)](https://user-images.githubusercontent.com/31947215/194193019-b46617ee-2009-42f2-a8da-4759532b1392.png)

**Comment by guarddog13** [comment]
Got it thank you!

**Comment by tteck** [comment]
Did everything work out ok?
Here's a rundown of what happened
1. The download path changed. They added a `-win.zip` file so I had to change the way the script was downloading the `.zip` so it wouldn't grab the unwanted file.
2. Had to move away from wildcard extraction.
3. The binary name changed so I had to redo the `zwave-js-ui.service` file and add that to the update script so it would continue to start on boot.
4. They added the creation of the directory `store` so the update has to be done at root now and files copied to `/opt/zwave-js-ui` so that `store` isn't overwritten 

Sorry it took me a bit to get it all worked out.
